src/pages/using-spark/components/button.mdx

```diff
@@ -18,29 +19,42 @@ An interactive element that represents an action or choice.
 
 ### Usage
 
-Use Buttons to represent the main actions on a page.
+Use buttons to represent what type of actions the users can take on a page and to create a visual hierarchy between different calls to action.
+
+> As a general rule, use no more than two primary call-to-action buttons ([Primary Button](#primary-button)) per page.
 
 Examples of Button usage:
-- To trigger a new UI element to appear on the page.
+- To trigger action.
 - To submit a form.
 - To begin a new task.
 - To navigate to the next step in a process.
+- To confirm completion or to exit out of a flow.
+
+If the action will take take the user to a new page or to an external site, consider using a [Link](/using-spark/components/link) instead.
 
 ### Guidelines
 
-- Button Text
-  - Use precise, task-specific action verbs like: “Add”, “Remove”, “Read Details”, and “Submit”.
-  - Avoid ambiguous words like “Yes”, “No”, “OK”, and “Click Here”.
-  - Use title case (capitalize the first letter each word)
-  - Use less than 20 characters.
-  - Never wrap to a second line.
-- Button width should not exceed 280px.
-- Do not use buttons in paragraphs or within a lines of text. Use [Link](/using-spark/components/link) instead.
-- Use Button variants to create action hierarchy within a page. Variant names imply their placement within that hierarchy.
-  - [Primary Button](#primary-button)
-  - [Secondary Button](#secondary-button)
-  - [Tertiary Button](#tertiary-button)
-  - [Quaternary Button](#quaternary-button)
+#### How to Write Effective Button Text
+- Button text should set clear expectation of what will happen on click.
```

Comment:
'should set a clear'
?

Reply:
yes you're right here, I'm going to change it to "clear expectation*s*"